docs/source/ecf.rst

```diff
@@ -540,16 +497,48 @@ The path to the directory in which to output the Stage 5 JWST data and plots.
 Stage 5 Fit Parameters
 ----------------------
 
-This file describes the transit/eclipse parameters and their prior distributions. Each line describes a new parameter, with the following basic format:
+.. warning::
+   The Stage 5 fit parameter file has the file extension ``.epf``, not ``.ecf``. These have different formats, and are not interchangeable.
+
+This file describes the transit/eclipse and systematics parameters and their prior distributions. Each line describes a new parameter, with the following basic format:
 
 ``Name    Value    Free    PriorPar1    PriorPar2    PriorType``
 
-The ``PriorType`` can be U (Uniform), LU (Log Uniform), or N (Normal). If U/LU, then ``PriorPar1`` and ``PriorPar2`` are the lower and upper limits of the prior distribution. If N, then ``PriorPar1`` is the mean and ``PriorPar2`` is the stadard deviation of the Gaussian prior.
+``Name`` defines the specific parameter being fit for. Available options are:
+   - Transit and Eclipse Parameters
+      - ``rp`` - planet radius, for the transit models.
```

Comment:
This should be the planet/star radius ratio

Reply:
Fixed.